gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/AlertNotifierServiceProxy.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.management.service;
+
+import io.gravitee.am.identityprovider.api.User;
+import io.gravitee.am.model.ReferenceType;
+import io.gravitee.am.model.alert.AlertNotifier;
+import io.gravitee.am.repository.management.api.search.AlertNotifierCriteria;
+import io.gravitee.am.service.model.NewAlertNotifier;
+import io.gravitee.am.service.model.PatchAlertNotifier;
+import io.reactivex.Completable;
+import io.reactivex.Flowable;
+import io.reactivex.Single;
+
+/**
+ * @author Rémi SULTAN (remi.sultan at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public interface AlertNotifierServiceProxy {
```

Comment:
I wondering if we have to redefine all methods for Proxy interfaces, why not just inherit from the service interface ?
`AlertNotifierServiceProxy extends AlertNotifierService`

Wdyt?

Reply:
I guess this makes sense.
Having a separated interface makes the separation of concerns more distinct. 
However I believe that whenever we expand a new method, it will be both use in the gateway and the management. 

If we start in the future to have a lot of methods only used in the service and not in the proxy for management, we'll have to rethink that choice.

In the meantime, that sounds like a good idea.